content/kubeone/master/infrastructure/production_recommendations.en.md

```diff
@@ -65,3 +65,13 @@ cluster_name    = "my-cool-cluster"
 ami_id          = "ami-00f6fb16625871821"
 internal_api_lb = true
 ```
+
+## Azure
+
+### internal Loadbalancer alternative
+
+In order to hide your Kubernetes API endpoint from the external world, it's recommended to use an internal loadbalancer. 
+However due the [limitations](https://docs.microsoft.com/en-us/azure/load-balancer/components#limitations) of the 
+loadbalancer, backend VM cannot call the frontend of it's loadbalancer. We recommend to use
+ [GoBetween Load Balancer][gobetween] as an alternative here. You will find an example in the 
```

Comment:
We definitely don't recommend gobetween on Azure :D

Reply:
Ok, so what do we recommend? 
The standard LB does not work with only a private IP

Reply:
I can see that there is private_ip_address config option in the frontend_ip_configuration of the [LB resource](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/lb#frontend_ip_configuration)

Reply:
removed as discussed